app/models/article.rb

```diff
@@ -535,9 +546,11 @@ def canonical_url_must_not_have_spaces
     errors.add(:canonical_url, "must not have spaces") if canonical_url.to_s.match?(/[[:space:]]/)
   end
 
+  # TODO: (Alex Smith) refactor liquid tag permissions
```

Comment:
Agreed! I thought about doing that at first and I left it to my name since all of this is just for the sake of iterating with smaller PR sizes. Once it's "done", any remaining TODOs I'll change to `@thepracticaldev/delightful`. Under the current format, it's easy for me to `grep` for my name and catch what I need quickly as I'm iterating.